src/containers/StreamDetails/index.tsx

```diff
@@ -0,0 +1,163 @@
+
+import receiveLogo from '/public/images/receive.svg';
+import copyLogo from '/public/images/whiteCopy.svg';
+import shareLogo from '/public/images/share.svg';
+import withdrawLogo from '/public/images/withdrawSolid.svg';
+
+const StreamDetails = () => {
+  const StreamDetailsTitle = (
+    <div className="w-full flex justify-between items-center pb-2">
+      <h1 className="text-[24px] text-midnightBlue pl-2 mt-2">Stream #100065</h1>
+      <CStreamDetailsStatus type="Active" />
+    </div>
+  );
+
+  const summaryTitle = (
+    <div className="w-full flex justify-between items-center pb-4 pl-4">
+      <h1 className="text-2xl text-midnightBlue">Summary</h1>
+    </div>
+  );
+
+  const statusTitle = (
+    <div className="w-full flex justify-between items-center pb-4 pl-4">
+      <h1 className="text-2xl text-midnightBlue">Status</h1>
+      <CButton
+        variant="simple"
+        color="outline"
+        content="Withdraw"
+        logo={withdrawLogo}
+        className="!px-3 !py-2 h-[40px]"
+      />
+    </div>
+  );
+
+  return (
+    <div className="w-full flex gap-4">
+      <CPageCard
+        divider
+        title={StreamDetailsTitle}
+        className="w-full px-6 py-[15px]"
+        childrenClassName="!pl-0"
+      >
+        <section className="flex flex-col items-center justify-center">
+          <div className="flex justify-center mb-6 mt-8">
+            <Image src={receiveLogo} alt="receiveLogo" height={0} width={0} />
+          </div>
+
+          <div className="flex flex-col items-center justify-center">
+            <h2 className="text-[40px]">+523.348 USDC</h2>
+            <p className="text-base">Total amount streamed</p>
+          </div>
+          <div className="w-[420px] mt-[32px]">
+            <CCard
+              className="flex flex-col justify-center items-center w-full h-full px-3 py-4"
+              bgColor="royalBlue"
+              borderColor="rgba(0, 0, 0, 0.10)"
+            >
+              <div className="flex justify-between items-center w-full">
+                <div className="flex justify-center items-center h-12 w-[53px] bg-[#442cd6] text-white text-base px-2 py-2.5 rounded-[9px]">
+                  60%
+                </div>
+
+                <div className="border border-white w-[85%] rounded-xl bg-[#442cd6] pl-[-5px]">
+                  <svg
+                    xmlns="http://www.w3.org/2000/svg"
+                    width="202"
+                    height="48"
+                    viewBox="0 0 202 48"
+                    fill="none"
+                  >
+                    <path
+                      d="M0.824463 10C0.824463 4.47715 5.30162 0 10.8245 0H201.824V48H10.8245C5.30161 48 0.824463 43.5229 0.824463 38V10Z"
+                      fill="white"
+                    />
+                    <path
+                      d="M0.824463 10C0.824463 4.47715 5.30162 0 10.8245 0H201.824V48H10.8245C5.30161 48 0.824463 43.5229 0.824463 38V10Z"
```

Comment:
move the SVG code to another file called StreamProgress

<StreamProgress progress={0 to 100}